app/views/sponsors/index.html.erb

```diff
@@ -32,10 +32,10 @@
         <% @sponsors.each do |sponsor| %>
           <tr>
             <td><%= sponsor.id %></td>
-            <td><%= image_tag sponsor.logo, height: '33' %></td>
+            <td><%= image_tag sponsor.logo, class: "logo" %></td>
```

Comment:
We can also resize logo to show in table.
look how resizing works with Activestorage